src/main/java/org/elasticsearch/index/query/MultiMatchQueryBuilder.java

```diff
@@ -165,6 +165,14 @@ public MultiMatchQueryBuilder(Object text, String... fields) {
     }
 
     /**
+     * ppc
```

Comment:
Thats strange, I think I already corrected and pushed that. Will check and redo if needed